Brocfile.js

```diff
@@ -1,6 +1,8 @@
 var merge = require('broccoli-merge-trees');
 var compileSass = require('broccoli-sass');
 
+ var includepaths = [ 'assets', 'bower_components', 'bower_components/fontawesome' 'bower_components/neat/app/assets/stylesheets' 'bower_components/app/assets/stylesheets' ]
```

Comment:
You are missing commas between the strings in your array. This will lead to a compile error when you try to run `broccoli-lr`